scalar/scalar
# feat(cli): `scalar check` command to check `scalar.config.json` files

This PR adds the `check` cli command to validate `scalar.config.json` files using [`typebox`](https://github.com/sinclairzx81/typebox) schema validation

## Example usage
```bash 
scalar check /packages/cli/src/commands/check/valid.json
```
![Screenshot 2024-09-25 at 12 09 21 PM](https://github.com/user-attachments/assets/fc99cc26-6b75-4952-9562-ee19ac7c5d26)
![Screenshot 2024-09-25 at 12 08 58 PM](https://github.com/user-attachments/assets/cbf63f94-8617-47cf-b87b-41b1ec65089a)
![Screenshot 2024-09-25 at 12 08 41 PM](https://github.com/user-attachments/assets/8570b885-3430-4c74-b005-b424a081bd32)






### To Test
You can use the ` pnpm --filter cli cli:link` command to check out these latest changes locally
